pom.xml

```diff
@@ -1,272 +1,305 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="http://maven.apache.org/POM/4.0.0"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.bynder</groupId>
-  <artifactId>bynder-java-sdk</artifactId>
-  <version>2.2.7</version>
-  <packaging>jar</packaging>
+    <groupId>io.github.arpit-sharma-usc</groupId>
+    <artifactId>byndertest-java-sdk</artifactId>
```

Comment:
Almost didn't notice this one, because of the indentation changes.. But I think we'll want to restore these two lines ;)

Reply:
oh no this wasn't there earlier on. I add it test with the test-account I created and then revert it back to bynder Ids. See [here](https://github.com/Bynder/bynder-java-sdk/pull/43/commits/c823f7e33b1b9279c698192a9e6df94d7d8ba86c) 😄